mutation/frozen_mutation.hh

```diff
@@ -232,6 +232,11 @@ public:
 
 frozen_mutation freeze(const mutation& m);
 std::vector<frozen_mutation> freeze(const std::vector<mutation>&);
+// Caller is responsible for keeping the argument stable in memory
+// freeze_gently may yield only between mutations, threfore it's suitable for
+// a long vector of short mutations.
```

Comment:
Not sure if a comment like this is neccessary. This is true for virtually all future returning methods.

Reply:
The thing is that it's different than unfreeze_gently, which peers through the mutation and maybe yields in-between rows, so I think it's worth mentioning for the unsuspecting user.